src/SFA.DAS.AssessorService.Settings/WebConfiguration.cs

```diff
@@ -14,26 +12,20 @@ public class WebConfiguration : IWebConfiguration
 
         [JsonRequired] public NotificationsApiClientConfiguration NotificationsApiClientConfiguration { get; set; }
 
-        [JsonRequired] public CertificateDetails CertificateDetails { get; set; }
-
         [JsonRequired] public string AssessmentOrgsApiClientBaseUrl { get; set; }
 
         [JsonRequired] public string IfaApiClientBaseUrl { get; set; }
 
-        [JsonRequired] public string IFATemplateStorageConnectionString { get; set; }
+        [JsonRequired] public string IFATemplateStorageConnectionString { get; set; } 
 
         [JsonRequired] public string SqlConnectionString { get; set; }
 
-        public string SpecflowDBTestConnectionString { get; set; }
-
         [JsonRequired] public string SessionRedisConnectionString { get; set; }
-        [JsonRequired] public AuthSettings StaffAuthentication { get; set; }
+        
         [JsonRequired] public ClientApiAuthentication QnaApiAuthentication { get; set; }
         [JsonRequired] public string ServiceLink { get; set; }
         [JsonRequired] public DfeSignInConfig DfeSignIn { get; set; }
```

Comment:
When EPAO originally written the DfeSignIn was used for authentication; then a new service the [das-login-service] was built instead - there was talk about this being temporary - going back to the DfeSignIn or using an entirely different government sign-in; I think the use of the LoginService will be long term and this part of the configuration should be renamed to appropriately.